packages/shared/tailwind.config.js

```diff
@@ -134,6 +134,7 @@ module.exports = {
       'img-mobile': '280px',
       'rank-modal': 'calc(100vh - 5rem)',
       page: 'calc(100vh - 3.5rem)',
+      textarea: '292px',
```

Comment:
Where possible we prefer rem, so I think 18.25 should do.

Reply:
Also that the `textarea` is too general. Can we maybe make it `commentBox` or something similar?